converters/fromZigbee.js

```diff
@@ -830,6 +830,48 @@ const converters = {
             };
         },
     },
+    heiman_smoke_battery: {
+        cid: 'genPowerCfg',
+        type: 'attReport',
+        convert: (model, msg, publish, options) => {
+            const batt = msg.data.data.batteryPercentageRemaining;
+            const batt_low = msg.data.data.batteryAlarmState;
+            var results = {};
+            if (batt != null) {
+                const value = Math.round(batt/255.0*10000)/100;; // Out of 255
+                results['battery'] = value;
+            }
+            if (batt_low != null) {
+                if (batt_low) {
+                    results['battery_low'] = true;
+                } else {
+                  results['battery_low'] = false;
+                }
+            }
+            return results;
+        },
+    },
+    heiman_smoke_battery2: {
+        cid: 'genPowerCfg',
+        type: 'devChange',
```

Comment:
I think this converters is not needed, AFAIK `devChange` is an event generated by zigbee-shepherd, it always comes together with an `attReport`.

Reply:
If that is indeed the case, you can replace it with an ignore converter.

Reply:
Yes, I believe it is. I always got two messages when I defined the report. One attReport and one devChange, which is why I added the "battery2" to avoid the "undefined converter"-message.

How do I add an "ignore converter"?

Reply:
Never mind, found it in the code. Will update.

Reply:
Oops. I also cut-and-paste to the wrong converter. My bad.